### PR TITLE
Landing page filter,product navigation logout

### PR DESCRIPTION
**Pull request by shlok30**
Hi Team,

I have added the following features in this branch

1)Filter for category gets applied by clicking on the category on the homepage

https://user-images.githubusercontent.com/60795588/161695852-d7ffc0e2-b2e4-4c4c-ab4e-082f12ba297c.mov

2)Added Product List Link on the Nav Bar for easy navigation

3)Added Logout functionality

https://user-images.githubusercontent.com/60795588/161696293-baa47288-9d7f-4cc6-90a2-832381190d14.mov

Preview Link : https://water-comm-react-git-landingpagefilterproductnav-5b4b3b-shlok30.vercel.app/

Can you please review this pull request.

Thanks in advance

### CODE REVIEW

File: my-app/src/pages/cart.js [hunk 1]
```diff
@@ -10,8 +10,8 @@ import { useUser } from '../context/user-context'
 
 const CartPage = () => {
     const {state:{cart}} = useUser()
-    
-    const renderCartCard = () => cart.map((prod) => <CartCard key = {prod['_id']} id = {prod['_id']} title = {prod.title} price = {prod.price} qty = {prod.qty} />)
+    console.log("From cart page",cart)
```
Comment (Rishi05051997) [inline my-app/src/pages/cart.js:13]: remove console

File: my-app/src/pages/login.js [hunk 2]
```diff
@@ -13,14 +13,12 @@ const LoginPage = () => {
...
-    console.log("User Context" , state )
-
     const handleClick = () => {
-        console.log({email,password})
+        
         axios
          .post("/api/auth/login",{email,password})
          .then(res => {
-            console.log("It was a success",res)
+            console.log("Login success, here is the user data in db",res.data)
```
Comment (Rishi05051997) [inline my-app/src/pages/login.js]: remove console

**Review by Rishi05051997 — APPROVED** [review]
left some comments , once its done then good to go

File: my-app/src/context/product-listing-context.js [hunk 3]
```diff
@@ -5,6 +5,7 @@ const ProductListingContext = createContext()
 
 const ProductListingContextProvider = ({children}) => {
     const [state , dispatch] = useReducer(productListReducer,{category : {fiction : true , 'non-fiction' : true , horror: true} , rating : "" , sorting : "" , priceRange : 1000})
```
Comment (ark2002) [inline my-app/src/context/product-listing-context.js:7]: maybe  give the initial state as a variable

**Review by ark2002 — APPROVED** [review]
the code looks good to go

**Pull request merged**